meetings/tool/src/main/resources/card_es.properties

```diff
@@ -0,0 +1,18 @@
+decription_link_text=Ver descripci\u00F3n
```

Comment:
Please provide _CA and _EU files for this.

Reply:
done